### PR TITLE
Bump slicedimage version.

### PR DESCRIPTION
**Pull request by ttung**
Some minor changes were required to get things to run.

### CODE REVIEW

**Comment by ambrosejcarr** [comment]
I am OK with you pushing your own version upgrades to master. No need for review requests, I think. I appreciate it must be annoying to wait for me, and I don't see any need. 👍

**Comment by ttung** [comment]
> I am OK with you pushing your own version upgrades to master. No need for review requests, I think. I appreciate it must be annoying to wait for me, and I don't see any need. 👍

Well, it turns out this breaks tests in hilarious ways, so I guess it's good I PR'ed it.

**Comment by ambrosejcarr** [comment]
> Well, it turns out this breaks tests in hilarious ways, so I guess it's good I PR'ed it.

Just noticed. x(

File: REQUIREMENTS.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ scikit-image>=0.14.0
 scikit-learn
 scipy
 showit >= 1.1.4
-slicedimage == 0.0.1
+slicedimage == 0.0.3
```
Comment (joshmoore) [inline REQUIREMENTS.txt]: Shouldn't this be 0.0.2?

Comment (ttung) [reply REQUIREMENTS.txt]: Nope, found some issues in 0.0.2 that necessitated some bugfixes.

**Pull request merged**